README.md

```diff
@@ -1,31 +1,32 @@
 # Rust RFCs
-[Rust RFCs]: #rust-rfcs
+[Opening]: #rust-rfcs
 
 Many changes, including bug fixes and documentation improvements can be
 implemented and reviewed via the normal GitHub pull request workflow.
 
-Some changes though are "substantial", and we ask that these be put
-through a bit of a design process and produce a consensus among the Rust
-community and the [sub-team]s.
+Some changes though are "substantial", and we ask that these be put through a
+bit of a design process and produce a consensus among the Rust community and
+the [sub-team]s.
 
-The "RFC" (request for comments) process is intended to provide a
-consistent and controlled path for new features to enter the language
-and standard libraries, so that all stakeholders can be confident about
-the direction the language is evolving in.
+The "RFC" (request for comments) process is intended to provide a consistent
+and controlled path for new features to enter the language and standard
+libraries, so that all stakeholders can be confident about the direction the
+language is evolving in.
 
 ## Table of Contents
 [Table of Contents]: #table-of-contents
-* [Opening](#rust-rfcs)
-* [Table of Contents]
-* [When you need to follow this process]
-* [Before creating an RFC]
-* [What the process is]
-* [The role of the shepherd]
-* [The RFC life-cycle]
-* [Reviewing RFC's]
-* [Implementing an RFC]
-* [RFC Postponement]
-* [Help this is all too informal!]
+
+  - [Opening]
+  - [Table of Contents]
+  - [When you need to follow this process]
+  - [Before creating an RFC]
+  - [What the process is]
+  - [The role of the shepherd]
+  - [The RFC life-cycle]
+  - [Reviewing RFCs]
+  - [Implementing an RFC]
+  - [RFC Postponement]
+  - [Help this is all too informal!]
```

Comment:
There were more instances - within the `README.md` - that are using `-` (hyphen) than there are using `*` (asterisk) so I made them all consistent.

If there is a preference for `*` I am happy to change them all to that. I was only aiming for consistency; not a particular character.